packages/@aws-cdk/aws-appsync/test/integ.log-retention.ts

```diff
@@ -0,0 +1,23 @@
+import { join } from 'path';
```

Comment:
Yay!!! I see you got the integration tests working! Looks perfect here.

Reply:
not really, I used `--dry-run` switch ... which doesn't deploy them to an account 😄

Reply:
the problem seems to be that the `integ-runner` doesn't pick my profile/region configuration when it is trying to deploy the stack.

Reply:
it does for the "snapshotting" – synth: `Running in parallel across profiles cloudgardener-dev-admin and regions: eu-west-1`

but not for the deploy: `AppSyncIntegLogRetention failed: Error: This stack uses assets, so the toolkit stack must be deployed to the environment (Run "cdk bootstrap aws://unknown-account/unknown-region")`

Reply:
I assume you have run `cdk bootstrap` for this account/region before and that in your CloudFormation stacks you have one called `cdkToolkit`?

Reply:
It seems that `integ-runner` works when using long-lived credentials, and default bootstrap with `cdk bootstrap` that result `CDKToolkit` CloudFormation stack. When using profile with `SSO` credentials or bootstrap that is deployed by some other mean e.g. CloudFormation StackSets ... it doesn't work.